avd_docs/aws/cloudtrail/AVD-AWS-0016/CloudFormation.md

```diff
@@ -3,7 +3,7 @@ Turn on log validation for Cloudtrail
 
 ```yaml---
 Resources:
-  BadExample:
+  GoodExample:
```

Comment:
ditto

Reply:
Similar to my previous comment, the policy script has been updated as part of this PR and this should say GoodExample:
https://github.com/aquasecurity/defsec/blob/master/rules/cloud/policies/aws/cloudtrail/enable_log_validation.cf.go